src/components/userProfile/components/SideMenu.js

```diff
@@ -17,9 +17,9 @@ export default class SideMenu extends React.Component {
                 mode={INLINE}
             >
                 {
-                    SIDE_MENU_ITEMS.map(sideMenuItem => (
-                        <Menu.Item key={sideMenuItem.label}>
-                            <span><Icon type={sideMenuItem.iconType} />{sideMenuItem.label}</span>
+                    SIDE_MENU_ITEMS.map(({ path, iconType, label }) => (
+                        <Menu.Item key={path}>
+                            <span><Icon type={iconType} />{label}</span>
```

Comment:
<span><Icon type={iconType} />{label}</span>
iconType should be capitalized if its a constant. 
if it is edit call it EDIT